oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -696,6 +725,14 @@ public Collection<String> getDefaultScopes() {
       return defaultScopes;
     }
 
+    public String getTransport() {
```

Comment:
Please provide javadoc on these, pointing to publicly documented Metadata Service parameters.

Reply:
Discussed offline: We will update public MDS docs when the feature is non-experimental. Since the usage of the S2A + MTLS + bound tokens flow is guarded by an environment variable (EXPERIMENTAL_GOOGLE_API_USE_S2A). Until the public docs are updated, we can link to AIP here and https://github.com/googleapis/google-auth-library-java/pull/1400 (AIP already linked in this merged PR).